contentcuration/contentcuration/views/internal.py

```diff
@@ -79,8 +79,8 @@ class NodeValidationError(ValidationError):
     pass
 
 
-def handle_server_error(request):
-    client.captureException(stack=True, tags={"url": request.path})
+def handle_server_error(e, request):
+    capture_exception(e, tags={"url": request.path})
```

Comment:
I'm having trouble identifying whether `tags` is supported here. The documentation details [a different example](https://docs.sentry.io/platforms/python/enriching-events/scopes/#using-with-scope)

Reply:
Yeah, I had to go to the source code to verify.

Ultimately it gets set on a transient scope using this function: https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/scope.py#L451